validator/client/validator.go

```diff
@@ -201,6 +202,7 @@ func (v *validator) UpdateAssignments(ctx context.Context, slot uint64) error {
 	// Only log the full assignments output on epoch start to be less verbose.
 	if slot%params.BeaconConfig().SlotsPerEpoch == 0 {
 		for _, assignment := range v.assignments.Assignment {
+			log.Error("2")
```

Comment:
and this?

Reply:
that was testing it out in runtime, just removed the logs. They are no longer there